notes/README.md

```diff
@@ -0,0 +1,59 @@
+
+# Notes
+
+A JavaFX application that uses Gluon Mobile Glisten (a mobile application framework with mobile-friendly UI controls) and Gluon Mobile Connect (for locally storing notes and user settings).
+
+
+## Documentation
+
+Read how to create this sample step by step [here](http://docs.gluonhq.com/samples/notes/)
+
+## Quick Instructions
+
+Please follow the pre-requisites as stated [here](https://github.com/gluonhq/client-samples/#build-and-run-the-samples).
```

Comment:
We add one more sentence before this line:

> We use [Gluon Client](https://docs.gluonhq.com/client/) to build a native image for platforms including desktop, android and iOS. Please follow the prerequisites as stated [here](https://github.com/gluonhq/client-samples/#build-and-run-the-samples).

Reply:
Ok, updated